SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,55 @@
+header:
+  schema-version: 0.13.4
+  last-updated: '2023-09-05'
+  last-reviewed: '2023-09-05'
+  expiration-date: '2024-09-05T01:00:00.000Z'
+  project-url: https://github.com/WasmEdge/WasmEdge
+  project-release: '0.13.4'
+project-lifecycle:
+  stage: active
+  bug-fixes-only: false
+  core-maintainers:
+  - github:juntao
+  - github:hydai
+  - github:q82419
+  - github:ibmibmibm
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+  code-of-conduct: https://github.com/WasmEdge/WasmEdge/blob/master/docs/CODE_OF_CONDUCT.md
```

Comment:
We also need to add `automated-tools-list`. https://github.com/ossf/security-insights-spec/blob/main/specification.md#contribution-policy